src/main/java/net/imglib2/img/display/imagej/ImageProcessorUtils.java

```diff
@@ -73,4 +75,17 @@ public static boolean isSupported( final Type< ? > type )
 		return ( type instanceof UnsignedByteType ) || ( type instanceof UnsignedShortType ) ||
 				( type instanceof ARGBType ) || ( type instanceof FloatType );
 	}
+
+	public static Img< ? > initArrayImg( final int sizeX, final int sizeY, final Object pixels )
```

Comment:
Please add javadoc (not clear from the signatory that `Object pixels` means primitive type array).

(I know that "no javadoc" keeps with the theme of existing imglib2-ij code. But let's try to improve this situation.)